src/pretix/base/templatetags/rich_text.py

```diff
@@ -54,6 +54,19 @@
 
 register = template.Library()
 
+
+def build_fediverse_re(tlds):
+    return re.compile(
+        r"""\(*  # Match any opening parentheses.
+        @[^@]+@
+        ([\w-]+\.)+(?:{0})(?:\:[0-9]+)?(?!\.\w)\b   # xx.yy.tld(:##)?
+        """.format(
+            "|".join(sorted(tlds))
```

Comment:
Is there a reason we sort the tlds again? They get sorted when passed via the lambda-call.